bin/node/runtime/src/lib.rs

```diff
@@ -667,7 +667,11 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type RewardHandler = (); // nothing to do upon rewards
 	type DataProvider = Staking;
 	type Solution = NposSolution16;
-	type Fallback = pallet_election_provider_multi_phase::NoFallback<Self>;
+	type Fallback = frame_election_provider_support::onchain::BoundedOnChainSequentialPhragmen<
+		Self,
+		frame_support::traits::ConstU32<10>,
+		frame_support::traits::ConstU32<100_000>,
```

Comment:
```suggestion
		frame_support::traits::ConstU32<20_000>,
		frame_support::traits::ConstU32<2_000>,
```
I don't think these value actually matter here, but I figure something closer to what we have to live chains might be a strong example.